google-cloud-tools-plugin/testSrc/com/google/cloud/tools/intellij/gcs/GcsBucketContentEditorPanelTest.java

```diff
@@ -36,51 +40,97 @@
 /** Tests for {@link GcsBucketContentEditorPanel}. */
 public class GcsBucketContentEditorPanelTest {
   @Rule public final CloudToolsRule cloudToolsRule = new CloudToolsRule(this);
+  @Rule public final TimeZoneRule timeZoneRule = new TimeZoneRule(TimeZone.getTimeZone("GMT"));
 
   private GcsBucketContentEditorPanel editorPanel;
   private GcsBucketVirtualFile bucketVirtualFile;
-  @Mock private Blob blob;
+
+  @Mock private Blob directoryBlob;
+  @Mock private Blob binaryBlob;
+
+  private BiMap<Integer, String> indexToColName =
+      HashBiMap.create(ImmutableMap.of(0, "Name", 1, "Size", 2, "Type", 3, "Last Modified"));
+  private Map<String, Integer> colNameToIndex = indexToColName.inverse();
 
   @Before
   public void setUp() {
     bucketVirtualFile = GcsTestUtils.createVirtualFileWithBucketMocks();
+
+    when(directoryBlob.isDirectory()).thenReturn(true);
+    when(directoryBlob.getName()).thenReturn("my_directory");
```

Comment:
Maybe pull these into static constants so writing tests below don't require you to read the `setUp()` method every time to remember what these values are?

Reply:
done